### PR TITLE
feat: make marker configurable

### PR DESCRIPTION
**Pull request by ArzelaAscoIi**
- make marker configurable to allow for multiple comments
- add marker as argument for action

### CODE REVIEW

File: src/requirements.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 diff-cover
-jinja2
```
Comment (ewjoachim) [inline src/requirements.txt:2]: Can you spare a word why you did that ? Was it necessary ? Could we solve the issue in a cleaner way ?

File: action.yml [hunk 2]
```diff
@@ -55,8 +55,8 @@ inputs:
     required: false
 runs:
   using: docker
-  # image: Dockerfile
-  image: docker://ewjoachim/coverage-comment-action:latest
+  image: Dockerfile
+  # image: docker://ewjoachim/coverage-comment-action:latest
```
Comment (ewjoachim) [inline action.yml:59]: Why ?

**Review by ewjoachim — COMMENTED** [review]
Thanks for your contribution ! The main change seems great. Would you mind rebasing your commits, and removing unrelated changes (or at least explaining why you had to make them)

(if you don't know how to rebase, no problem, just tell me, I'll do it and/or explain)

**Comment by ArzelaAscoIi** [comment]
Thanks for the review! I have to admit that i messed it up a little 🙃 I did the same changes + some tests [here](https://github.com/py-cov-action/python-coverage-comment-action/pull/180). 

Closing this PR

File: action.yml (hunk 2 above)
Comment (ArzelaAscoIi) [reply action.yml:59]: Was just for testing and forgot about the PR. Sorry!

**Comment by ewjoachim** [comment]
Ok ! Let's handle that in the other action. I hope it's clear that those are 2 different GitHub actions 😅

**Comment by ArzelaAscoIi** [comment]
> Ok ! Let's handle that in the other action. I hope it's clear that those are 2 different GitHub actions 😅

I know now 😅 but was too deep into running/debugging than checking that there is also an action with  a `python-` and thats the one I need to have a look at. Sorry for this 🙃

**Pull request closed without merging**